tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -172,3 +172,9 @@ spec:
         jq '.message // "Unset"' <<< "${conditions}" | tee $(results.requestMessage.path)
 
         jq -r '.indexImageDigests' <<< "${results}" |  tee $(results.indexImageDigests.path)
+
+        # shellcheck disable=SC3011
```

Comment:
I think we said passing checkton is not required for existing tasks for now. But if you want to make it right, why not just move to `bash` in the shebang? `<<<` is not POSIX compliant

Reply:
BTW, the mocks file has `bash` already and ideally it should always use the same as the actual task, otherwise your testing is diverging from the real thing. (I think the reason was that POSIX doesn't support `-` in function names.)